code/modules/hydroponics/grown/seedling.dm

```diff
@@ -0,0 +1,28 @@
+/obj/item/seeds/seedling
+	name = "pack of seedling seeds"
+	desc = "They grow into a helpful plant specie that can help look after plants!"
```

Comment:
```suggestion
	desc = "These seeds grow into a floral assistant which can help look after other plants!"
```
`specie` is still wrong and `These seeds grow into` seems to be the standard opening.
Putting on my english teacher hat I also didn't like having both `helpful` and `help` in the same sentence 😅 
really just changing `specie` to `species` would be sufficient though if you like it